pennylane/transforms/__init__.py

```diff
@@ -12,46 +12,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This subpackage contains QNode, quantum function, device, and tape transforms.
+This subpackage contains PennyLane transforms and their building blocks.
 
 
 .. currentmodule:: pennylane
 
-Transforms
-----------
+Custom transforms
+-----------------
 
-Transforms that act on QNodes
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+:class:`pennylane.transform` can be used to define custom transformations
+that work with PennyLane QNodes; such transformations can map a circuit
+to one or many new circuits alongside associated classical post-processing.
 
-These transforms accept QNodes, and return new transformed functions
-that compute the desired quantity.
+See the :class:`pennylane.transform` documentation for more details and examples.
 
 .. autosummary::
     :toctree: api
 
-    ~transforms.classical_jacobian
-    ~batch_params
-    ~batch_input
-    ~batch_partial
-    ~metric_tensor
-    ~adjoint_metric_tensor
-    ~specs
-    ~transforms.split_non_commuting
+    ~transforms.core.transform
 
-Transforms that act on quantum functions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Transforms developer functions
+------------------------------
```

Comment:
@rmoyard please could we move this section to the bottom?

Reply:
Yes done!